tools/c7n_azure/c7n_azure/query.py

```diff
@@ -78,8 +60,10 @@ def __new__(cls, name, parents, attrs):
             attrs['filter_registry'] = FilterRegistry(
                 '%s.filters' % name.lower())
         if 'action_registry' not in attrs:
-            attrs['action_registry'] = ActionRegistry(
+            actions = ActionRegistry(
                 '%s.actions' % name.lower())
+            actions.register('tag', Tag)
```

Comment:
fwiw we also have some registry events/subscribers for adding filters/actions out of band, for unconditional registration this fine though.

Reply:
Sweet. I did see that being done for some AWS actions. Tags should be supported in all of our ARM resources, so I think it is fine to apply to all resources for now.